ton_client/src/net/endpoint.rs

```diff
@@ -40,12 +93,17 @@ impl Endpoint {
         format!("{}/graphql", base_url.trim_end_matches("/"))
     }
 
-    pub async fn resolve(client_env: Arc<ClientEnv>, address: &str) -> ClientResult<Self> {
+    pub async fn resolve(
+        client_env: Arc<ClientEnv>,
+        config: &NetworkConfig,
+        address: &str,
+    ) -> ClientResult<Self> {
         let address = Self::expand_address(address);
         let start = client_env.now_ms() as i64;
+        let query = "?query=%7Binfo%7Bversion%20time%20lastBlockTime%7D%7D";
```

Comment:
Current implementation is not compatible with q-server without `lastBlockTime` support. Maybe we should make them compatible for the first time?